aea/aea.py

```diff
@@ -89,6 +90,8 @@ def __init__(
         connection_exception_policy: ExceptionPolicyEnum = ExceptionPolicyEnum.propagate,
         loop_mode: Optional[str] = None,
         runtime_mode: Optional[str] = None,
+        default_ledger: Optional[str] = None,
```

Comment:
At some point we need to separate the config for the AEA from that of the context here. But this PR does not do that.